src/app.js

```diff
@@ -56,7 +56,7 @@ app.use("/api/auth", AuthController);
 app.use("/api/records", RecordController);
 app.use("/api/posts", PostController);
 
-require('./cronJob.js');
+//require('./cronJob.js');
```

Comment:
This part was commented for development purpose as it was interrupting the development server.
However, once it is available to be deployed, it needs to be reactivated.